extensions/skins/conversation_v1/player.html

```diff
@@ -30,6 +30,35 @@
                    ng-class="{'conversation-skin-progress-dots-iframed': isIframed, 'conversation-skin-progress-dots-not-iframed': !isIframed}">
     </progress-dots>
 
+    <div ng-if="helpHtml" class="conversation-skin-help-card-container">
+      <div class="conversation-skin-help-card">
```

Comment:
Just a note, it seems to me that we could greatly benefit from having a directive for a conversation-skin-card, which can be reused for the help card, the main tutor card, and the future tutor card. At the moment, this template has numerous copies of essentially the same thing.